webapp/components/CommentList/CommentList.vue

```diff
@@ -50,3 +49,15 @@ export default {
   },
 }
 </script>
+
+<style lang="scss">
+.comment-list {
+  > .title {
+    margin-top: 0;
+
+    > .counter-icon {
+      margin-right: $space-small;
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-14T17:13:54Z" title="Tuesday, January 14th 2020, 6:13:54 pm +01:00">Jan 14, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -50,3 +49,15 @@ export default {
   },
 }
 </script>
+
+<style lang="scss">
+.comment-list {
+  > .title {
+    margin-top: 0;
+
+    > .counter-icon {
+      margin-right: 12px;
```


I thought we wanted to avoid using magic numbers instead of design tokens, @alina-beck ??

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-15T07:42:01Z" title="Wednesday, January 15th 2020, 8:42:01 am +01:00">Jan 15, 2020</time>_

---


got me... :blush: we do!